_tags

```diff
@@ -4,9 +4,9 @@ true: warn(@5@8@10@11@12@14@23-24@26@29@40), bin_annot, safe_string, debug
 <node_modules/**>: -traverse
 "formatTest": -traverse
 "src": include
-<src/*>: package(menhirLib unix compiler-libs.common ocamlbuild findlib easy-format str BetterErrors merlin_extend re.str)
-<src/reason_utop.*>: package(menhirLib utop str re.str)
+<src/*>: package(menhirLib unix compiler-libs.common ocamlbuild findlib easy-format BetterErrors merlin_extend re.str)
```

Comment:
I wonder if we need to remove `re.str`. The build has been failing due to not being able to find `str` from within  the pretty printer code.